### PR TITLE
Week 2: Understand Fetch API and implement a simple quoteServlet

### PR DESCRIPTION
**Pull request by karngyan**
# Description

- Added a new endpoint '/quotes' to the servlets. Fetching the quotes from server instead of hard coding it for typewriter effect.
- Added a favicon

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/QuotesServlet.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        quotes.add("It’s going to happen, because I am going to make it happen.");
+        quotes.add("It’s not bragging if it’s true.");
+        quotes.add("Win a no win situation by rewriting the rules.");
+        quotes.add("Let them hate, just make sure they spell your name right.");
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        String quotesJsonString = new Gson().toJson(quotes);
```
Comment (arijit-google-sps) [inline portfolio/src/main/java/com/google/sps/servlets/QuotesServlet.java:39]: These quotes are simple strings. Why do you want to use json?

Comment (arijit-google-sps) [inline portfolio/src/main/java/com/google/sps/servlets/QuotesServlet.java]: These quotes are simple strings. Why do you want to use json?

Comment (karngyan) [reply portfolio/src/main/java/com/google/sps/servlets/QuotesServlet.java]: It's supposed to be an array of strings. I didn't want to process too much on client side. 

Are there any easier way to do this? text/plain or text/html requires me too process it using a separator may be.

PS. Also merged the next step of JSON in the walkthrough.

Comment (arijitakacool) [reply portfolio/src/main/java/com/google/sps/servlets/QuotesServlet.java]: Yes, thanks for merging the JSON part here.
I meant could be better if you find a better suitable use case for json. For example, in the walkthrough, you'll see a parent field people in the json, and each element under it has a name and birthyear.
Maybe we can think of a good use-case for our example too, where we can use some key-value structures and jsonify that.

Comment (karngyan) [reply portfolio/src/main/java/com/google/sps/servlets/QuotesServlet.java]: Sure. On it.

Comment (karngyan) [reply portfolio/src/main/java/com/google/sps/servlets/QuotesServlet.java]: Hey @arijit-google-sps @arijitakacool

Re-requesting review.

**Review by arijit-google-sps — APPROVED** [review]

**Pull request merged**